controllers/apply/awards-for-all/fields.test.js

```diff
@@ -288,79 +288,6 @@ describe('fields', () => {
         });
     }
 
-    function testContactDateOfBirth(field, minAge) {
```

Comment:
Moved these tests to `form.test.js`, aiming to migrate all these tests over to `form.test.js` soon.